warehouse/accounts/views.py

```diff
@@ -178,7 +178,7 @@ def accounts_search(request) -> dict[str, list[User]]:
     Used with autocomplete.
     User must be logged in.
     """
-    if request.authenticated_userid is None:
+    if request.identity is None:
```

Comment:
Nitpick: maybe use `request.user` here and below? The AuthN/Z layers should prevent `request.identity` from being anything except a `User` in this context, but accessing through `user` is slightly more self-describing.

Reply:
I like that, and will give it a shot. However, if I'm reading the intent correctly, this kind of change **should not** be applied to the Macaroon policy, since that isn't always a User, correct?

Reply:
Correct!

Reply:
I was able to convert over most usages to `request.user` except for some of the totp/webauthn because of how the tests are currently constructed.